docs/source/host/quick-start.rst

```diff
@@ -0,0 +1,59 @@
+########################
+Hosting Quick Start
+########################
+
+.. contents:: On This Page
+    :depth: 2
+    
+----------------------------
+Notice: Port Forwarding
+----------------------------
+
+There are two ways to make a domain accessible from the outside Internet.
+
+1. You may register your domain with the Metaverse server by creating a `Domain ID` in the domain's web panel found at http://localhost:40100/.
+2. Alternatively, you can manually open the following ports on your router and firewall:
+
+    * 40100 : (tcp) administration web portal
+    * 40102 : (udp) main connection for clients
+
+----------------------------
+Windows
+----------------------------
+
+1. Download Vircadia (full, not Interface only).
+2. Use the `Express` install option, or if you use the `Custom` option ensure that both the `Vircadia Interface` and `Vircadia Sandbox` options are checked. Vircadia Sandbox is the domain server.
```

Comment:
Has this not been renamed to Vircadia Server yet? (I am not on Windows so I wouldn't know)